eth/gasprice/feehistory.go

```diff
@@ -97,12 +97,12 @@ func (oracle *Oracle) processBlock(bf *blockFees, percentiles []float64) {
 
 	// Fill in blob base fee and next blob base fee.
 	if excessBlobGas := bf.header.ExcessBlobGas; excessBlobGas != nil {
-		blobBaseFee256, err := misc.GetBlobGasPrice(chainconfig, *excessBlobGas)
+		blobBaseFee256, err := misc.GetBlobGasPrice(chainconfig, *excessBlobGas, bf.header.Time)
 		if err != nil {
 			bf.err = err
 			return
 		}
-		nextBlobBaseFee256, err := misc.GetBlobGasPrice(chainconfig, misc.CalcExcessBlobGas(chainconfig, bf.header))
+		nextBlobBaseFee256, err := misc.GetBlobGasPrice(chainconfig, misc.CalcExcessBlobGas(chainconfig, bf.header), bf.header.Time)
```

Comment:
Should be the time of the next header here (`bf.header.Time+SecondsPerSlot` approximately)